Makefile

```diff
@@ -277,6 +290,15 @@ $(GOLANGCILINT):
 $(MISSPELL):
 	$(call fetch_go_bin_version,github.com/client9/misspell/cmd/misspell,$(MISSPELL_VERSION))
 
+$(ALERTMANAGER):
+	$(call fetch_go_bin_version,github.com/prometheus/alertmanager/cmd/alertmanager,$(ALERTMANAGER_VERSION))
+
+$(MINO_SERVER):
```

Comment:
This variable is never defined, which means that the target isn't executed and minio isn't installed :/

Reply:
:man_facepalming:

Reply:
fixed

Reply:
here the variable is called `MINO_SERVER` and up top it's defined as `MINIO_SERVER` so we have the same issue. We just need to fix the misspelling here for it to work

Reply:
fixed v2.0

Reply:
:sunglasses: